Do not force static link, it breaks build with
# BR2_STATIC_LIBS is not set
BR2_SHARED_LIBS=y

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

diff -uNr libseccomp-2.2.0.org/tests/Makefile.am libseccomp-2.2.0/tests/Makefile.am
--- libseccomp-2.2.0.org/tests/Makefile.am	2015-02-11 21:57:11.000000000 +0100
+++ libseccomp-2.2.0/tests/Makefile.am	2015-03-29 16:03:49.668946652 +0200
@@ -16,7 +16,6 @@
 # along with this library; if not, see <http://www.gnu.org/licenses>.
 #
 
-AM_LDFLAGS = -static
 LDADD = util.la ../src/libseccomp.la
 
 check_LTLIBRARIES = util.la
diff -uNr libseccomp-2.2.0.org/tools/Makefile.am libseccomp-2.2.0/tools/Makefile.am
--- libseccomp-2.2.0.org/tools/Makefile.am	2015-02-11 21:57:11.000000000 +0100
+++ libseccomp-2.2.0/tools/Makefile.am	2015-03-29 16:03:46.164992302 +0200
@@ -33,8 +33,6 @@
 scmp_bpf_sim_SOURCES = scmp_bpf_sim.c bpf.h util.h
 
 scmp_sys_resolver_LDADD = ../src/libseccomp.la
-scmp_sys_resolver_LDFLAGS = -static
 scmp_arch_detect_LDADD = ../src/libseccomp.la
-scmp_arch_detect_LDFLAGS = -static
 scmp_bpf_disasm_LDADD = util.la
 scmp_bpf_sim_LDADD = util.la
